src/main/java/uk/gov/justice/hmpps/prison/service/enteringandleaving/ReleasePrisonerService.kt

```diff
@@ -38,7 +38,7 @@ class ReleasePrisonerService(
       request.movementReasonCode,
       booking,
       toLocation,
-      request.commentText,
+      request.commentText ?: "",
```

Comment:
Is there a restriction at the client side that means it can't be nullable?

Reply:
In the schema the property is nullable so no reason for the client's to make it non-nullable.